modules/android/vsandroid_vcxproj.lua

```diff
@@ -95,42 +119,40 @@
 
 	function android.androidAPILevel(cfg)
 		if cfg.androidapilevel ~= nil then
-			_p(2,'<AndroidAPILevel>android-%d</AndroidAPILevel>', cfg.androidapilevel)
+			vc2010.element("AndroidAPILevel", nil, "android-" .. cfg.androidapilevel)
 		end
 	end
 
 	function android.androidStlType(cfg)
 		if cfg.stl ~= nil then
 			local stlType = {
-				["none"] = "none",
-				["minimal c++ (system)"] = "system",
-				["c++ static"] = "gabi++_static",
-				["c++ shared"] = "gabi++_shared",
-				["stlport static"] = "stlport_static",
-				["stlport shared"] = "stlport_shared",
-				["gnu stl static"] = "gnustl_static",
-				["gnu stl shared"] = "gnustl_shared",
-				["llvm libc++ static"] = "c++_static",
-				["llvm libc++ shared"] = "c++_shared",
+				none = "system",
+				["gabi++"] = "gabi++",
+				stlport = "stlport",
+				gnustl = "gnustl",
+				["libc++"] = "c++",
```

Comment:
Can you fix the style so that it is consistent? ["..."] is missing from 'none', 'stlport' and 'gnustl'

Reply:
[Done](https://github.com/premake/premake-core/pull/1264/commits/b01f495a258218e3d56f57807c029056b5c5a44f)